common-items.lic

```diff
@@ -34,7 +34,7 @@ module DRCI
 
   def dispose(item, trash_room = nil)
     DRCT.walk_to(trash_room) unless trash_room.nil?
-    DRC.bput("get my #{item}", 'You get')
+    DRC.bput("get my #{item}", 'You get', 'What were you')
```

Comment:
If the result of the 'get' is 'What were you', should we return early? I.e. is there any point in calling `dispose_trash`?